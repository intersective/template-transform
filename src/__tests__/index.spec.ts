import { transform } from '../index';

it('1. get the settings of a user', async () => {
  const result = transform('1', '1', "{'name': 'Practera'}");
  expect(result).toMatchSnapshot();
});

it('2. parsing an email', async () => {
  const result = transform('em', 'ail', {message: {body: {html: '<i>Hello World</i>', text: 'Hello World'}}});
  expect(result).toMatchSnapshot();
});

it('3. parsing a sms', async () => {
  const result = transform('s', 'ms', {message: {title: 'sweet', body: 'dude'}});
  expect(result).toMatchSnapshot();
});

it('4. parsing a fasdf', async () => {
  const result = transform('fas', 'df', {message: {title: 'dude', body: 'sweet'}});
  expect(result).toMatchSnapshot();
});
