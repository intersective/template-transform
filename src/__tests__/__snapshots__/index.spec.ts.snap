// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`1. get the settings of a user 1`] = `"{'name': 'Practera'}"`;

exports[`2. parsing an email 1`] = `
Object {
  "contentsHtml": "<i>Hello World</i>",
  "contentsText": "Hello World",
}
`;

exports[`3. parsing a sms 1`] = `
Object {
  "contents": "dude",
  "title": "sweet",
}
`;

exports[`4. parsing a fasdf 1`] = `Object {}`;
